g2p/convert.py

```diff
@@ -16,15 +16,18 @@ class BlissLexiconToG2PLexiconJob(Job):
     Convert a bliss lexicon into a g2p compatible lexicon for training
     """
 
-    def __init__(self, bliss_lexicon, all_phons=False):
+    __sis_hash_exclude = {"write_first_phoneme_only": True}
+
+    def __init__(self, bliss_lexicon, write_first_phoneme_only=True):
```

Comment:
Here we are considering the different pronunciations and this terminology (phoneme, writing the first phoneme) is not clear. I would call this boolean `include_pronunciation_variants`

Reply:
Can we all agree on this before I change again @mmz33 @JackTemaki?

Reply:
@curufinwe ? I would agree to Tinas suggestion.

Reply:
I also agree with Tina's suggestion.